### PR TITLE
Upgrade MathJax to 2.7.4 and enable SRE for accessibility

### PR DESCRIPTION
**Pull request by pumazi**
Upgrade MathJax to 2.7.4 and enable SRE for accessibility. MathJax was upgraded because the MathJax SRE extension doesn't exist for MathJax 2.6.1.

The [AssistiveMML](https://docs.mathjax.org/en/latest/options/extensions/assistive-mml.html) just exposes MathML to browsers that support reading it... it does not add human-readable alt-text... the [mathjax-sre.js](https://github.com/mathjax/MathJax-a11y/blob/master/docs/README.md#introduction) does that (adds human-readable text).

In order to include `mathjax-sre.js` we had to upgrade MathJax to 2.7

### CODE REVIEW

**Comment by philschatz** [comment]
@pumazi Unfortunately, I'm having trouble finding the `aria-label` attribute on Math elements... maybe we _do_ need the link for how to resolve `[a11y]/mathjax-sre.js` after all (& maybe explicitly disable `AssistiveMathML`

**Comment by reedstrm** [comment]
ok. Though, the actual _contents_ of the `aria-label` seem … obscure? "collapsed separator separated list of length 1 period" ???

**Comment by philschatz** [comment]
@reedstrm yeah. "Simple" math does seem fine though:

image


The abbreviated alt  text is probably because of the [collapsible a11y plugin](https://docs.mathjax.org/en/latest/options/extensions/a11y-extensions.html#auto-collapse-js) which is enabled by default. We _could_ disable it but I think it's probably enabled so people can skip over long math and explore more if they want.

**Comment by reedstrm** [comment]
Ah, so may actually be usable by the intended audience, then.
